rpcd/playbooks/maas-20151013-1-requirements.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Part of MaaS Migration: maas-20151013.yml
+- name: Ensure MaaS Migration Requirements
+  hosts: hosts
+  user: root
+  tasks:
+    - name: Ensure migrations index exists
+      local_action: file path={{ migrations_dir }} state=directory
+      when: inventory_hostname == groups['hosts'][0]
+    - name: Locate evidence of prior MaaS migration
+      local_action: stat path={{ migrations_dir }}/maas-20151013
```

Comment:
I asked @stevelle the same question when I was reviewing these initially. It's meant as a version number, e.g., this migration was written on 2015-10-13 and is named as such so that other migrations can be written an used.